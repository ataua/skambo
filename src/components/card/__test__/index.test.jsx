import React from 'react';
import Card from '../index';
import TestRenderer from 'react-test-renderer';

describe('Card  Snapshot test', () => {
  it('Card Snapshot test Render', () => {
    const tree = TestRenderer.create(<Card />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});