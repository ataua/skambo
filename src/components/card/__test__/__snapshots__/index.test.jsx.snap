// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card  Snapshot test Card Snapshot test Render 1`] = `
<div
  style={Object {}}
>
  <div
    className="sc-bdnylx qAWFh"
  >
    <img
      alt="card"
    />
    <div>
      <strong
        onBlur={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
      />
      <p />
    </div>
  </div>
</div>
`;
